### PR TITLE
Adapt ufld model to tensorrt8

### PR DESCRIPTION
**Pull request by tsieyy**
I changed some source code ，and adapt ufld model to tensorrt8
Now，it works fine in my machine.

I use command `./lane_det -s` it genereted lane.engine successfully, and I also use command `./lane_det -d ./files/` to test
, it works fine too!
## env

 **jetson nano**
Jetpack: 4.6.1
- Cuda: 10.2.300
- cuDNN: 8.2.1.32
- TensorRT: 8.2.1.8
- OpenCV: 4.5.5
- Python: 3.7.12

### CODE REVIEW

**Comment by wang-xinyu** [comment]
@tsieyy Thanks for the PR. We can make it compatible with both TRT7 and TRT8, can you please check the logging.h and macro.h in yolov5 of this repo?

File: ufld/CMakeLists.txt [hunk 1]
```diff
@@ -12,6 +12,11 @@ find_package(CUDA REQUIRED)
 
 include_directories(${PROJECT_SOURCE_DIR}/include)
 
+# cuda_runtime_api
+include_directories(/usr/local/cuda-10.2/targets/aarch64-linux/include)
```
Comment (wang-xinyu) [inline ufld/CMakeLists.txt:16]: Here is only for aarch64, we need remove these 2 lines.

Comment (tsieyy) [reply ufld/CMakeLists.txt]: fine，I have removed them.

**Comment by tsieyy** [comment]
I refer to the writing method of the unet model, have modified the code according to your requirements, and tested it on my machine, at least it can run normally in the environment of tensorrt8.

**Pull request merged**